en/configure-a-tidb-cluster.md

```diff
@@ -557,6 +557,20 @@ The YAML file above:
 
 When Kubernetes deletes the TiDB Pod, it also removes the TiDB node from the service endpoints. This is to ensure that the new connection is not established to this TiDB node. However, because this process is asynchronous, you can make the system sleep for a few seconds before you send the `kill` signal, which makes sure that the TiDB node is removed from the endpoints.
 
+## Configure TiKV cluster graceful upgrade
+
+Before restart tikv, we will evict all leaders in it, we only restart it when the leader number drop into zero or reach a specified timeout.
```

Comment:
During TiKV upgrade, TiKV Pod is restarted only after the upgrade evicts all Region leaders on TiKV Pod (which means the Region leader number drops to 0) or the eviction reaches the specified timeout (10 minutes by default).